docs/api-client/README.md

```diff
@@ -0,0 +1,929 @@
+
+**Response**
+
+Returns `Promise<string>` - a promise which resolves with the ID of the uploaded contract.
+
+Example
+
+```
+
+```
+
+### getContract
+
+Get contract by its ID.
+
+```
+GET /contract/raw/:contractId
+```
+
+**Parameters**
+
+| Name | Type | Description |
+| ---- | ---- | ----------- |
+| `contractId` | `string` | **Required.** The ID of the contract. |
+
+Example
+
+```
+
+```
+
+**Response**
+
+Returns `Promise<Object>` - a promise which resolves with a contract object.
+
+Example
+
+```
+{
+
+}
+```
+
+### searchContracts
+
+List contracts based on search term.
+
+```
+GET /contracts/search
+```
+
+**Parameters**
+
+| Name | Type | Description |
+| ---- | ---- | ----------- |
+| `keyword` | `string` | **Required.** The search term to filter displayed contracts. |
+| `page` | `number` | **Required.** Which group of contracts to display. Each group consists of `pageSize` contracts. |
+| `pageSize` | `number` | **Required.** The number of contracts to display on page. |
+| `sortBy` | `string` | **Required.** The way in which returned contracts are to be sorted: `creationdate`, `privatename`, or `title` |
+
+Example
+
+```
+
+```
+
+**Response**
+
+Returns an array of contracts.
+
+Example
+
+```
+[
+
+]
+```
+
+### sendContract
+
+Send a contract to other users.
+
+```
+POST /send/contract
+```
+
+**Headers:** `'Content-Type': 'application/x-www-form-urlencoded'`
+
+**Parameters**
+
+| Name | Type | Description |
+| ---- | ---- | ----------- |
+| `readonlyEmails` | `Array<string>` | An array of users' emails who will have read-only access to the contract. |
+| `editEmails` | `Array<string>` | An array of users' emails who will have access to edit the contract.
+| `id` | `string` | **Required.** The ID of the contract to be sent.
+
+Example
+
+```
+
+```
+
+### changeContractAlias
+
+Change private name of contract.
+
+```
+GET /contract/alias/:contractId
+```
+
+**Parameters**
+
+| Name | Type | Description |
+| ---- | ---- | ----------- |
+| `contractId` | `string` | **Required.** The ID of the contract to be given a new private name. |
+| `newName` | `string` | **Required.** The new private name of the contract. |
+
+Example
+
+```
+
+```
+
+### stopContract
+
+Stop smart contract transactions that are pending for an executed contract.
+
+```
+GET /contract/stop/:id
+```
+
+**Parameters**
+
+| Name | Type | Description |
+| ---- | ---- | ----------- |
+| `id` | `string` | **Required.** The ID of the contract with the pending smart contract transactions. |
+
+Example
+
+```
+
+```
+
+### resumeContract
+
+Resume scheduled execution of smart contract transactions that were stopped for an executed contract.
+
+```
+GET /contract/resume/:id
+```
+
+**Parameters**
+
+| Name | Type | Description |
+| ---- | ---- | ----------- |
+| `id` | `string` | **Required.** The ID of the contract with the stopped smart contract transactions. |
+
+Example
+
+```
+
+```
+
+### uploadContractToGoogle
+
+Upload a contract to user's Google Drive.
```

Comment:
This feature is not merged yet pending some UX fixes, should we leave this out for now?

Reply:
Sure we can remove for now.